eZ/Publish/API/Repository/Tests/SearchService/SearchServiceFulltextEmbedTest.php

```diff
@@ -0,0 +1,184 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\API\Repository\Tests\SearchService;
+
+use eZ\Publish\API\Repository\SearchService;
+use eZ\Publish\API\Repository\Tests\BaseTest;
+use eZ\Publish\API\Repository\Values\Content\LocationQuery;
+use eZ\Publish\API\Repository\Values\Content\Query;
+use eZ\Publish\API\Repository\Values\Content\Query\Criterion;
+use eZ\Publish\API\Repository\Values\Content\Search\SearchHit;
+use eZ\Publish\Core\FieldType\RichText\Value as RichTextValue;
+use eZ\Publish\Core\Repository\Values\Content\Content;
+use eZ\Publish\Core\Repository\Values\Content\ContentCreateStruct;
+
+/**
+ * Test case for full text search in the SearchService (for embed).
+ *
+ * @see \eZ\Publish\API\Repository\SearchService
+ * @group integration
+ * @group search
+ * @group fulltext
+ */
+class SearchServiceFulltextEmbedTest extends BaseTest
+{
+    private const EMBEDDED_ARTICLE_NAME = 'test1';
+
+    private static $createdIds = [];
+
+    protected function setUp()
+    {
+        parent::setUp();
+
+        $repository = $this->getRepository(false);
+
+        if (false === $repository->getSearchService()->supports(SearchService::CAPABILITY_ADVANCED_FULLTEXT)) {
```

Comment:
readability, the line is too long:
``` php
        if (
            false === $repository->getSearchService()->supports(
                SearchService::CAPABILITY_ADVANCED_FULLTEXT
            )
        ) {
            $this->markTestSkipped(
                'Advanced FullText search is not supported by the current search engine'
            );
        }
```
or
``` php
        if ($this->searchServiceSupportsAdvancedFullTextSearch()) {
            $this->markTestSkipped(
                'Advanced FullText search is not supported by the current search engine'
            );
        }
```
where is a `protected` method of the `BaseTest`.

Reply:
There is no method searchServiceSupportsAdvancedFullTextSearch in BaseTest and nothing similar, so I'll made a change with first example